cli/js/ops/fs/open.ts

```diff
@@ -29,9 +29,14 @@ export interface OpenOptions {
    * access to be used. When createNew is set to `true`, create and truncate
    * are ignored. */
   createNew?: boolean;
+  /** Permissions to use if creating the file (defaults to `0o666`, before
+   * the process's umask).
+   * It's an error to specify mode without also setting create or createNew to `true`.
+   * Raises on Windows. */
```

Comment:
Mode should be ignored on windows - and not result in error.

@piscisaureus correct?